components/card/card-footer-link.js

```diff
@@ -151,26 +128,26 @@ class CardFooterLink extends RtlMixin(LitElement) {
 				target="${ifDefined(this.target)}"
 				type="${ifDefined(this.type)}">
 				<span class="d2l-offscreen">${this.text}</span>
+				<d2l-count-badge-icon
+					aria-hidden="true"
+					icon="${this.icon}"
+					max-digits="${ifDefined(this.secondaryCountMaxDigits ? this.secondaryCountMaxDigits : undefined)}"
+					number="${noNumber ? 0 : this.secondaryCount}" 
+					?hide-zero="${noNumber}"
+					text="${this.text}"
+					type="${this.secondaryCountType}">
```

Comment:
Not sure why the linter is complaining at this line, saying that type accepts only 'count' or 'notification', which I set the definition to in this file.